--- wxWidgets-3.1.4/include/wx/defs.h.orig.h	2021-01-18 15:37:11.874087400 +0000
+++ wxWidgets-3.1.4/include/wx/defs.h	2021-01-18 15:35:08.928139900 +0000
@@ -389,7 +389,7 @@ typedef short int WXTYPE;
         #define HAVE_STD_STRING_COMPARE
     #elif defined(__VISUALC__)
         #define HAVE_STD_STRING_COMPARE
-    #elif defined(__MINGW32__) || defined(__CYGWIN32__)
+    #elif defined(__MINGW32__) || defined(__CYGWIN__)
         #define HAVE_STD_STRING_COMPARE
     #endif
 #endif
